ci_framework/roles/operator_build/molecule/default/converge.yml

```diff
@@ -21,27 +21,49 @@
     cifmw_operator_build_dryrun: true
     cifmw_operator_build_push_ct: false
   tasks:
-    - name: Build mariadb-operator
+    - name: Build keystone-operator
       include_role:
         name: operator_build
       vars:
         cifmw_operator_build_meta_build: true
         cifmw_operator_build_operators:
-          - name: "mariadb-operator"
-            src: "{{ ansible_user_dir }}/mariadb-operator"
+          - name: "keystone-operator"
+            src: "{{ ansible_user_dir }}/keystone-operator"
 
     - name: Ensure that role output dict contains the expected values
       ansible.builtin.assert:
         that:
           # Check the explicit build operator
-          - cifmw_operator_build_output['operators']['mariadb-operator']['commit_hash'] is defined
-          - cifmw_operator_build_output['operators']['mariadb-operator']['image'] is defined
-          - cifmw_operator_build_output['operators']['mariadb-operator']['image_bundle'] is defined
-          - cifmw_operator_build_output['operators']['mariadb-operator']['image_storage_bundle'] is defined
-          - cifmw_operator_build_output['operators']['mariadb-operator']['image_catalog'] is defined
+          - cifmw_operator_build_output['operators']['keystone-operator']['commit_hash'] is defined
+          - cifmw_operator_build_output['operators']['keystone-operator']['image'] is defined
+          - cifmw_operator_build_output['operators']['keystone-operator']['image_bundle'] is defined
+          - cifmw_operator_build_output['operators']['keystone-operator']['image_storage_bundle'] is defined
+          - cifmw_operator_build_output['operators']['keystone-operator']['image_catalog'] is defined
           # Check the meta operator build output
           - cifmw_operator_build_output['operators']['openstack-operator']['commit_hash'] is defined
           - cifmw_operator_build_output['operators']['openstack-operator']['image'] is defined
           - cifmw_operator_build_output['operators']['openstack-operator']['image_bundle'] is defined
           - cifmw_operator_build_output['operators']['openstack-operator']['image_storage_bundle'] is defined
           - cifmw_operator_build_output['operators']['openstack-operator']['image_catalog'] is defined
+
+    - name: Build mariadb-operator from Zuul job
```

Comment:
Care to assert we get the related content in the cifmw_operator_build_output? Just to be on the safest possible side